README.md

```diff
@@ -44,6 +46,17 @@ https://github.com/customerio/customerio-ios.git
 
 ![](docs/img/xcode_spm_select_sdks.jpeg)
 
+### Available SDK packages
+
+Here are the list of the SDKs that you can install. You can find more details on both in [our SDK documentation](/docs/sdk/ios/).
```

Comment:
> You can find more details on both in [our SDK documentation](/docs/sdk/ios/)

I am wondering if this provides any value to the docs here? When I read docs for products, I find it annoying when someone says, "Learn more" and sends me to the landing page where I am expected to browse my way to where I need to be. 

One idea could be that instead of having a generic "learn more about all the products" here, we instead make `Tracking`, `MessagingPush`, etc text in the table below link to each page in the docs talking about that product?

Reply:
That's fair. I should probably also alias some /docs/ pages to to the name of the package.